ccm_web/server/src/main.ts

```diff
@@ -83,9 +85,11 @@ async function bootstrap (): Promise<void> {
     SwaggerModule.setup('swagger', app, document)
   }
 
+  if (!isDev) app.enableShutdownHooks(['SIGINT'])
+
   await app.listen(
     serverConfig.port,
-    () => logger.info(`Server started on localhost and port ${serverConfig.port}`)
+    () => logger.info(`Server started on 0.0.0.0 and port ${serverConfig.port}`)
```

Comment:
I looked at the Openshift setup log and this statement is repeated 4 time. Why is this behavior?

Reply:
Well, `pm2` starts up four different server instances, so these log statements will show up 4 times. I could add a process ID to the log messages using `winston` I think, though I wasn't sure if that was needed.

Reply:
does pm2 starts 4 instance by default? since I did not see a configuration around it?

Reply:
See `ecosystem.config.js`. We are using `cluster` mode and a `max` setting for instances. 4 seems to be what OpenShift pods want to go to so far.

Reply:
ok